runtime/near-vm-logic/src/gas_counter.rs

```diff
@@ -103,8 +100,7 @@ impl GasCounter {
             self.fast_counter.burnt_gas.checked_add(burn_gas).ok_or(HostError::IntegerOverflow)?;
         let new_used_gas =
             new_burnt_gas.checked_add(new_promises_gas).ok_or(HostError::IntegerOverflow)?;
-        if new_burnt_gas <= self.max_gas_burnt && (self.is_view || new_used_gas <= self.prepaid_gas)
-        {
+        if new_burnt_gas <= self.max_gas_burnt && new_used_gas <= self.prepaid_gas {
```

Comment:
can you add the comment in line 94 - to explain what this method does and what are the arguments?
(what is the difference between 'burn' gas and 'use' gas?)

Reply:
Added some short docs. Problem is that I’m not completely clear on how the gas accounting work.